src/prelude.rs

```diff
@@ -9,7 +9,6 @@ pub use embedded_hal::prelude::*;
     not(any(feature = "stm32f411", feature = "stm32f412", feature = "stm32f401",))
 ))]
 pub use crate::dac::DacExt as _stm32f4xx_hal_dac_DacExt;
-pub use crate::dma::DmaExt as _stm32f4xx_hal_dma_DmaExt;
```

Comment:
Is this usable without re-exporting the traits in the prelude?

Reply:
This trait don't exist anymore, I changed to using the `fn new()` approach, the user doesn't need any trait in scope to use the API, they might need the `Stream` trait to use some helper methods, but it isn't usually required.